pkg/email/templates/client_requested.html.tmpl

```diff
@@ -0,0 +1,91 @@
+{{- define "title" -}}
+Client Requested
+{{- end -}}
+
+{{- define "preview" -}}
+Your review is required for a newly registered OAuth client.
+{{- end -}}
+
+{{- define "body" -}}
+<p>
+Dear {{ .ReceiverName }},
```

Comment:
If we use such addresses, then lets use them consistently throughout.

Reply:
We do.